SteamProject/MySteam/Category.cpp

```diff
@@ -6,23 +6,52 @@ FCategory::FCategory()
     GamesAmount = 0;
 }
 
-FCategory::FCategory(const std::string NewCategoryName)
+//-------------------------------------------------------------------------------------------------------------
+
+FCategory::FCategory(const std::string& NewCategoryName)
 {
     CategoryName = NewCategoryName;
     GamesAmount = 0;
 }
+
+FCategory::FCategory(const FCategory &InCategory)
+{
+    CategoryName = InCategory.CategoryName;
+    GamesAmount = InCategory.GamesAmount;
+
+    //Memory copie array
+    memcpy_s(GameArray, sizeof GameArray, InCategory.GameArray, sizeof InCategory.GameArray);
+
+    /*
+    for (int i=0; i < GamesAmount; i++)
```

Comment:
Nice use of memcpy_s :) I see you had your previous code as a loop and you decided to comment it, but keep in mind that you should not leave unused code in your code base. 

This pollutes your code base and as it increases in size and development time, these comments become confusing.